src/modules/Overview/modal/EditMaxVMInFlightModal.tsx

```diff
@@ -26,7 +26,15 @@ export const EditMaxVMInFlightModal: FC<EditSettingsModalProps> = (props) => {
 
   // Set default value to 20
   const { resource } = props;
-  resource.spec.controller_max_vm_inflight = resource.spec.controller_max_vm_inflight || 20;
+
+  if (
+    resource?.spec &&
+    typeof resource?.spec === 'object' &&
+    'controller_max_vm_inflight' in resource.spec
```

Comment:
can't find this field in [types](https://github.com/search?q=repo%3Akubev2v%2Fforklift-console-types%20controller_max_vm_inflight&type=code). Is it missing?

Reply:
dont know, but if u search in the project you see it :)